src/Scrutor/ServiceCollectionExtensions.Scanning.cs

```diff
@@ -32,6 +32,25 @@ public static IServiceCollection Scan(this IServiceCollection services, Action<I
             return services.Populate(selector, RegistrationStrategy.Append);
         }
 
+        public static IServiceCollection Scan(this IServiceCollection services, Action<ITypeSelector> action)
```

Comment:
in VS2017, it just recognizes if you use one or the other with intellisense magic.  It will combine the methods of both `ITypeSelector` and `IImplementationTypeSelector` in the list.

would you prefer to have a different name than `Scan` ?